### ISSUE TITLE
Running In Codespaces

### ISSUE THREAD

**Issue by souder14** [reworkd/AgentGPT#1113]
### Please check that this issue hasn't been reported before.

- [X] I searched previous [Bug Reports](https://github.com/reworkd/AgentGPT/labels/bug) didn't find any similar reports.

### Expected Behavior

Auto GPT should allow for me to sign in on codespaces. I ran it in docker on my desktop and it worked fine. 

### Current behaviour

I got the Aging GPT running in code spaces but Currently when I click sign in I'm presented with this error {"type":"https://tools.ietf.org/html/rfc7231#section-6.5.1","title":"Bad Request","status":400,"traceId":"00-359df2f4aa3fc69670cb3eab1d72260c-ace2b71aa3b27752-00"}

### Steps to reproduce

Go through the setup process. Open the Local Host URL. Click the sign in button. The error shows up. I have already attempted to make the port public. 

### Possible solution

I'm assuming it has something to do with the way code spaces handles localhost

### Which Operating Systems are you using?

- [ ] Android
- [ ] iPhone/iPad
- [ ] Linux
- [ ] macOS
- [X] Windows

### Acknowledgements

- [X] My issue title is concise, descriptive, and in title casing.
- [X] I have searched the existing issues to make sure this bug has not been reported yet.
- [X] I am using the latest version of AgentGPT.
- [X] I have provided enough information for the maintainers to reproduce and diagnose the issue.

**Comment by jasangill1** [comment]
Hello @souder14 Hey buddy not sure if that's a typo but this is agentGPT no autoGPT!

**Comment by souder14** [comment]
That was definitely a major typo

**Comment by jasangill1** [comment]
All good @souder14 how you tried a local download ?

**Comment by souder14** [comment]
Yes its just extremely slow when running on my laptop. I only have a 1st gen surface laptop.

**Comment by jasangill1** [comment]
@souder14 have you tried to use a VM?
